docs/source/basics.rst

```diff
@@ -3,202 +3,147 @@
 Programs and Gates
 ==================
 
-First, initialize a localQVM instance on your laptop. You should have two consoles open in your terminal to run in the
-background.
+.. note::
 
-.. code:: python
-
-    ### CONSOLE 1
-    $ quilc -S
-    port triggered: 6000.
-    [2018-09-19 11:22:37] Starting server: 0.0.0.0 : 6000.
+    If you're running locally, remember set up the QVM and quilc in server mode before trying to use
+    them: :ref:`server`.
 
-    ### CONSOLE 2
-    $ qvm -S
-    Welcome to the Rigetti QVM
-    (Configured with 2048 MiB of workspace and 8 workers.)
-    [2018-09-20 15:39:50] Starting server on port 5000.
+Introduction
+~~~~~~~~~~~~
 
-Quantum programs are written in Forest using the ``Program`` object from the ``quil`` module.
+Quantum programs are written in Forest using the ``Program`` object. This ``Program`` abstraction will help us
```

Comment:
can't wait for the bright and lustrous future where we can link to the api reference for this